src/app/directives/documentation.tpl.html

```diff
@@ -186,6 +186,33 @@ <h4 class="raml-console-resource-body-heading">
             <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[code][responseInfo[code].currentType].example)"></code></pre>
           </div>
 
+          <div ng-if="responseInfo[code][responseInfo[code].currentType].examples">
+            <span>Examples:</span>
+            <div ng-if="isArray(responseInfo[code][responseInfo[code].currentType].examples)">
```

Comment:
In this case it's somebody else responsibility to make sure data is consumable by template. Let's say there is requirement (contract) from template - it expects to receive an array of objects where each object has "name" and "content" property. Now we need to deal with following cases:

- when there is only "example" property, which is string, we need to convert it to array of one element with generated name
- when there is "examples" as array, we need to generate names
- when there is "examples" as object, we need to sort names and generate array